### PR TITLE
COR-588: files menu

### PR DESCRIPTION
**Pull request by internetti**
move file actions to navbar

### CODE REVIEW

File: web/templates/nav.gohtml [hunk 1]
```diff
@@ -11,12 +26,45 @@
             <div id="navbarContent">
                 <div class="me-auto mb-2 mb-lg-0 w-100">
                     <div class="d-flex flex-row w-100 align-items-center">
-                        <a href="/static/nrc_grf_template.xlsx" download="nrc-grf-template" class="nav-link">
-                            Download template
-                        </a>
 
                         <div class="btn-group ms-5">
-                            <button type="button" class="btn btn-link text-secondary dropdown-toggle" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            <button type="button" class="btn btn-link text-secondary dropdown-toggle px-2" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```
Comment (ludydoo) [inline web/templates/nav.gohtml:31]: This button is only valid for the individuals page. I think that we should rather add some javascript in the `individuals.gohtml` to dynamically create this element and insert it the navbar

**Comment by neb42** [comment]
Why are we moving this to the nav bar? The ux seems wrong here

File: web/templates/individuals.gohtml [hunk 2]
```diff
@@ -297,30 +316,21 @@
...
                             <span id="deleteIndividualsButtonContent">Delete individuals</span>
                         </button>
+
+                        <button onclick="downloadFunc()"
+                                style="width: 22rem"
+                                class="btn disabled btn-outline-secondary me-2 individual-action-button"
+                                id="downloadIndividualsButton">
+                            <i class="bi bi-download"></i>
+                            <span id="downloadIndividualsButtonContent">Download individuals</span>
+                        </button>
```
Comment (neb42) [inline web/templates/individuals.gohtml:326]: Should this still be here if it's being moved to the navbar?

File: web/templates/nav.gohtml [hunk 3]
```diff
@@ -11,26 +26,63 @@
...
+                                Files
+                            </button>
+                            <div class="dropdown-menu">
+                                 <a href="/static/nrc_grf_template.xlsx"
+                                    download="nrc-grf-template"
+                                    class="dropdown-item"
+                                 >
+                                    Download template
+                                </a>
+                                {{if .RequestContext.SelectedCountry}}
```
Comment (neb42) [inline web/templates/nav.gohtml:56]: This check needs to be more than "SelectedCountry". We need to check we are on the correct page

File: web/templates/individuals.gohtml (hunk 2 above)
Comment (internetti) [reply web/templates/individuals.gohtml:326]: this one has a different purpose. the navbar one downloads the filtered (if no filters are selected, all) individuals, this one downloads the manually selected individuals.
I agree that this is not optimal, but I do think that the button to download the selected individuals should not be in the navbar.

File: web/templates/nav.gohtml (hunk 3 above)
Comment (internetti) [reply web/templates/nav.gohtml:56]: but the correct permissions are checked in the next line. and we need to show something if no country was selected yet. what do you mean by correct page?

Comment (neb42) [reply web/templates/nav.gohtml:56]: The correct page is the individuals list. The download functionality doesn't make sense on any other page (which is why it is weird to have it in the navbar). The download template makes a bit more sense as it is constant.

**Pull request merged**